src/test/java/com/box/sdk/BoxMetadataCascadePolicyTest.java

```diff
@@ -0,0 +1,144 @@
+package com.box.sdk;
+
+import com.eclipsesource.json.JsonObject;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit.WireMockClassRule;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+/**
+ *
+ */
+public class BoxMetadataCascadePolicyTest {
+
+    /**
+     * Wiremock
+     */
+    @ClassRule
+    public static final WireMockClassRule WIRE_MOCK_CLASS_RULE = new WireMockClassRule(53621);
+    private BoxAPIConnection api = TestConfig.getAPIConnection();
+
+    @Test
+    @Category(UnitTest.class)
+    public void testCreateMetadataCascadePolicySucceedsSendsCorrectJson() throws IOException {
```

Comment:
We should also add a test for the `BoxFolder` instance method version of this call